graylog2-web-interface/src/pages/RulesPage.tsx

```diff
@@ -68,6 +73,10 @@ const RulesPage = () => {
     }
   }, [isPaginationReady, pagination]);
 
+  useEffect(() => {
+    _loadRuleMetricData(setIsMetricDataLoading, setMetricsConfig);
```

Comment:
When the user updates the debug metrics configuration using `RuleMetricsConfigContainer`, the change is not reflected until the page is reloaded. We should instead provide immediate feedback and re-fetch the metrics after the change has been done.